pkg/cluster/kubernetes_cluster.go

```diff
@@ -190,12 +174,87 @@ func (c *KubernetesCluster) Connect(ctx context.Context, cmdOptions *ConnectArgu
 		return err
 	}
 
-	err = c.createKafkaConnectionCustomResource(ctx, currentNamespace, &kafkaInstance)
+	switch cmdOptions.SelectedService {
+	case "kafka":
+		err = c.checkAndCreateKafkaConnectionCustomResource(ctx, currentNamespace, cmdOptions.SelectedServiceID)
+		if err != nil {
+			return err
+		}
+	case "service-registry":
+		err = c.checkAndCreateServiceRegistryConnectionCustomResource(ctx, currentNamespace, cmdOptions.SelectedServiceID)
+		if err != nil {
+			return err
+		}
+	case "":
```

Comment:
What is this case for? Assumption that kafka is default?
I will advice to not do it this way (now).

Logic should be:
- check if service type provided
- if not use interactive mode to ask for type
- case logic always check for type